libs/common-services/src/test/java/uk/gov/di/ipv/core/library/fixtures/VcFixtures.java

```diff
@@ -0,0 +1,890 @@
+                TestVc.TestCredentialSubject.builder()
+                        .address(null)
+                        .name(List.of((ALICE_PARKER_NAME)))
+                        .birthDate(List.of(new BirthDate("1970-01-01")))
+                        .passport(null)
+                        .build();
+        return generateVerifiableCredential(
+                TestVc.builder()
+                        .evidence(
+                                List.of(
+                                        TestVc.TestEvidence.builder()
+                                                .txn("e5b22348-c866-4b25-bb50-ca2106af7874")
+                                                .failedCheckDetails(
+                                                        List.of(Map.of("checkMethod", "data")))
+                                                .build()))
+                        .credentialSubject(credentialSubject)
+                        .build(),
+                "urn:uuid:51dfa9ac-8624-4b93-aa8f-99ed772ff0ec",
+                "https://review-xx.account.gov.uk",
+                Instant.ofEpochSecond(1697097326));
+    }
+
+    static String vcTicf() throws Exception {
+        return generateVerifiableCredential(
+                TestVc.builder()
+                        .credentialSubject(null)
+                        .evidence(TICF_EVIDENCE)
+                        .type(
+                                new String[] {
+                                    VERIFIABLE_CREDENTIAL_TYPE, RISK_ASSESSMENT_CREDENTIAL_TYPE
+                                })
+                        .build(),
+                "urn:uuid:d1823066-2137-4380-b0ba-4b61947e08e6",
+                "https://tcif.stubs.account.gov.uk",
+                Instant.ofEpochSecond(1704822570));
+    }
+
+    static String vcVerificationM1a() throws Exception {
+        TestVc.TestCredentialSubject credentialSubject =
+                TestVc.TestCredentialSubject.builder()
+                        .address(List.of(ADDRESS_4))
+                        .name(List.of((ALICE_PARKER_NAME)))
+                        .birthDate(List.of(new BirthDate("1970-01-01")))
+                        .passport(null)
+                        .build();
+        return generateVerifiableCredential(
+                TestVc.builder()
+                        .evidence(
+                                List.of(
+                                        TestVc.TestEvidence.builder()
+                                                .txn("abc1234")
+                                                .verificationScore(2)
+                                                .build()))
+                        .credentialSubject(credentialSubject)
+                        .build(),
+                "test-subject",
+                "https://review-k.integration.account.gov.uk",
+                Instant.ofEpochSecond(1653403140));
+    }
+
+    static String vcDcmawM1b() throws Exception {
```

Comment:
Is this signifcantly different to the driving permit VC above? Can they be reduced?